app/controllers/api/v1/reviews_controller.rb

```diff
@@ -5,11 +5,13 @@ class Api::V1::ReviewsController < ApiController
   def create
     review = Review.new(review_params)
     review.artist_id = artist.id
+    review.user_id = user.id
```

Comment:
in rails we can conver to 
`review.user = user`

But you shouldnt need a `user` method. Just use `current_user`!!!